common/src/main/java/com/microsoft/identity/common/internal/logging/ILoggerCallback.java

```diff
@@ -36,4 +39,4 @@
      * @param containsPII True if the log message contains PII, false otherwise.
      */
     void log(String tag, Logger.LogLevel logLevel, String message, boolean containsPII);
-}
+}
```

Comment:
can we add back new line ending?

Reply:
whoops

1f6548afc07d15e6fecbc1765f03de535febfe84